gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Dec 28 10:00:20 PST 2015
+#Fri May 12 17:06:15 EDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
I'll take your word for it that it's good to change this gradle version... I don't remember what went into picking 2.2.... make sure everything we do is right for Android 4.4.  Odd that in both the old and new code, this number doesn't match what is in build.gradle.